packages/core/i18n/nls.json

```diff
@@ -453,7 +453,14 @@
       "tabDefaultSize": "Specifies the default size for tabs.",
       "tabMaximize": "Controls whether to maximize tabs on double click.",
       "tabMinimumSize": "Specifies the minimum size for tabs.",
-      "tabShrinkToFit": "Shrink tabs to fit available space."
+      "tabShrinkToFit": "Shrink tabs to fit available space.",
+      "window": {
+        "tabCloseIconPlacement": {
+          "description": "Place the close icons on tab titles at the start or end of the tab. The default is the host OS convention.",
+          "end": "Place the close icon at the end of the label. In left-to-right languages, this is the right side of the tab.",
+          "start": "Place the close icon at the start of the label. In left-to-right languages, this is the left side of the tab."
+        }
+      }
```

Comment:
Please don't add the translations manually to the `nls.json` file. We have a workflow for that that automatically performs this task before the release. See also [here](https://github.com/eclipse-theia/theia/wiki/Internationalization-Updates#theia-translations).

Reply:
Thanks, Mark! I'll revert that.

Reply:
Reverted with commit f61560f4facc535f4c1e46a78e30025e67b4fefd. Perhaps that linked documentation can add a note that these files are not meant to be edited by hand when adding new translatable strings.